### DISCUSSION TITLE
forbid v-window to drag

### DISCUSSION THREAD

**Discussion by larspava** [vuetifyjs/vuetify#17193] [Q&A] [answered]
Hello,
I'm using vue 2 in a project and I used the v-window component to create a form where every v-window-item contains a field of this form with questions like name, age, and others. It has buttons with the word "back" and "next" to return or go to the next page, just like the ["costumized arrow"](https://v2.vuetifyjs.com/en/components/windows/#customized-arrows) example in the vuetify 2 page. These buttons call a function to verify if the current input was filled correctly.

My current problem is, the v-window allows the user to drag the page and get to the other v-window-item, without fill the previous field, ignoring the button verification.

How can I block this drag function from v-window? I looked the documentation and didn't find anything about this or didn't understand.

(didn't find any discussion about this, this is my first time starting one discussion, if I did wrong please let me know)
Thanks

**Comment by ziadbastawy** [comment]
@larspava  Did you discovered a solution for this issue because I'm struggling with it also.

**Reply by larspava** [reply, marked as answer]
Yes, I created an empty method that is called in the attempt of swipe . I don't know if it's the best method but it worked. 
Something like this:

on HTML:
```
<v-window v-model="step" :touch="touch">...
```
on JS:
```
data() {
   touch: {
                left: this.preventSwipe,
                right: this.preventSwipe
            }
...

 methods: {
        preventSwipe() {
        }, //empty method
```
